tests/integration/test_tls.py

```diff
@@ -47,7 +47,7 @@ async def check_tls_rewind(ops_test: OpsTest) -> None:
             logs = await run_command_on_unit(
                 ops_test,
                 unit.name,
-                "grep rewind /var/log/postgresql/postgresql.log",
+                "grep rewind /var/log/postgresql/postgresql-*.log",
```

Comment:
For the future: check it better as the current check does not guaranties a tls in use... it is just a rewind word on logs...